src/components/code-block/prism.worker.js

```diff
@@ -0,0 +1,9 @@
+import * as prism from '../../lib/prism';
+
+export function highlight(code, lang) {
+	if (lang == 'sh') lang = 'bash';
```

Comment:
Good catch 👍

Reply:
I'm actually not 100% sure if this is even needed. I couldn't find an alias for it in Prism, but we are also missing the `"shell"` lang definition there.